### PR TITLE
Build for android natively

### PR DESCRIPTION
**Pull request by randombenj**
Adds the `app/android` directory which enables to build an `.apk` file.

### CODE REVIEW

**Comment by eddex** [comment]
do the changes in the [app/.babelrc](https://github.com/randombenj/yelims/pull/2/commits/75ca383c0e038eaccdb5e3599613282b4c8ddedc#diff-f2747bbc6b50e5a7acd1e710bac98340) file disable the live preview with expo?

**Comment by randombenj** [comment]
Good point have to test it ...

**Comment by timofurrer** [comment]
What would we have to do to also support iOS?

**Comment by eddex** [comment]
> What would we have to do to also support iOS?

Buy a Mac, buy an iPhone, get an Apple developer account.. and then figure out how to do it :^)

File: app/android/app/src/main/AndroidManifest.xml [hunk 1]
```diff
@@ -1,29 +1,26 @@
 <manifest xmlns:android="http://schemas.android.com/apk/res/android"
-    package="com.airbnb.android.react.maps.example">
+    package="com.yelims">
```
Comment (fliiiix) [inline app/android/app/src/main/AndroidManifest.xml:2]: this is not ok :D

**Review by fliiiix — COMMENTED** [review]
ich hoffe ich komm am Wochenende mal dazu das zu probieren

**Review by fliiiix — CHANGES_REQUESTED** [review]
npm start doesn't work

**Pull request closed without merging**